src/config.ts

```diff
@@ -53,7 +60,7 @@ export const SUPPORTED_LEVELS = ['critical', 'error', 'warn', 'info', 'verbose',
 export type SupportedLevels = typeof SUPPORTED_LEVELS[number]
 
 export const DEFAULT_BEE_API_URL = 'http://localhost:1633'
-export const DEFAULT_HOST = '127.0.0.1'
+export const DEFAULT_HOST = 'localhost:3000'
```

Comment:
Probably don't want the port here, right?
```suggestion
export const DEFAULT_HOST = 'localhost'
```

Reply:
Well that was because of the confusions between HOST and HOSTNAMEs 😅 But yeah nope.